utils/passport.js

```diff
@@ -1,12 +1,14 @@
 import passport from 'passport';
-import { getUserByEmail } from './loginHelpers.js';
+import User from '../models/user.js';
 
 export const jwtCallback = (jwt_payload, done) => {
-    const user = getUserByEmail(jwt_payload.email);
-    if (user) {
-        return done(null, user);
-    }
-    return done(null, false);
+    User.findOne({ email: jwt_payload.email })
+        .then((user) => {
```

Comment:
I already mentioned about this condition) you can rewrite it one line. like this for example:
`done(null, user || false)`